packages/utils/src/deviceDetector/userAgentData.ts

```diff
@@ -0,0 +1,15 @@
+import { AgentInfo } from './types';
+
+export function parseUserAgentData(): AgentInfo {
+  const agent = navigator.userAgentData;
+  const brands = [...(agent.brands || agent.uaList)];
```

Comment:
```suggestion
  const [target] = [...(agent.brands || agent.uaList)];
```

처럼 구조분해할당을 사용하면 하단의 `brands[0]`을 제거할 수 있을 것 같습니다...!

Reply:
질문) 근데 이 값이 `null`이거나 `undefined`일 가능성은 전혀 없는건가유???